lib/libzutil/zutil_import.c

```diff
@@ -929,22 +1006,35 @@ zpool_read_label(int fd, nvlist_t **config, int *num_labels)
 
 	if (lio_listio(LIO_WAIT, aiocbps, VDEV_LABELS, NULL) != 0) {
 		int saved_errno = errno;
-
-		if (errno == EAGAIN || errno == EINTR || errno == EIO) {
-			/*
-			 * A portion of the requests may have been submitted.
-			 * Clean them up.
-			 */
-			for (l = 0; l < VDEV_LABELS; l++) {
-				errno = 0;
-				int r = aio_error(&aiocbs[l]);
-				if (r != EINVAL)
+		int tmpret = -1;
```

Comment:
We could reuse `error` instead of introducing a new return variable.

Reply:
Good point. Done!